templates/home/index.html

```diff
@@ -112,6 +142,8 @@ <h1 data-aos="zoom-in" data-aos-duration="1500">Jagrati Promotes <span style="co
     </section>
     <!-- Showcase ends -->
 
+    <button onclick="topFunction()" id="myBtn" title="Go to top" class="ml-5"><i class="fa fa-angle-double-up" aria-hidden="true" style="color: black;font-weight: bolder;"></i></button>
```

Comment:
Please put this button below the code for "Contact Us"  and above "Footer". Same thing for the CSS.